### PR TITLE
KAFKA-7740   Kafka Admin Client should be able to manage user/client …

### PR DESCRIPTION
**Pull request by ghost**
…configurations for users and clients

*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by omkreddy** [comment]
@yaodong66 Thanks for the PR. But we need to have KIP for this change.
https://cwiki.apache.org/confluence/display/KAFKA/Kafka+Improvement+Proposals

**Comment by ghost** [comment]
@omkreddy thanks a lot for looking at this PR. The KIP is here:
https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=97555704

File: core/src/main/scala/kafka/zk/AdminZkClient.scala [hunk 1]
```diff
@@ -303,7 +303,7 @@ class AdminZkClient(zkClient: KafkaZkClient) extends Logging {
       DynamicConfig.Client.validate(configs)
     else
       DynamicConfig.User.validate(configs)
-    changeEntityConfig(ConfigType.User, sanitizedEntityName, configs)
+      changeEntityConfig(ConfigType.User, sanitizedEntityName, configs)
```
Comment (stanislavkozlovski) [inline core/src/main/scala/kafka/zk/AdminZkClient.scala]: nit: this whitespace seems unneeded

**Comment by mtrienis** [comment]
Any ETA on when this might get merged in?

**Comment by cmccabe** [comment]
This was superseded by #8083

**Pull request closed without merging**